backend/app/controllers/campaigns_controller.rb

```diff
@@ -103,6 +103,7 @@ def create_or_update_campaign_charities
       campaign_charity = CampaignCharity.find_or_initialize_by(id: param[:id])
       campaign_charity.giving_sg_url = param[:giving_sg_url]
       campaign_charity.charity_id = param[:charity][:id]
+      campaign_charity.save!
```

Comment:
Let's not do `save!` here since we already do so on line 48, to make sure that the changes are atomic. 

Can we do the assignment to `@campaign.campaign_charities` on line 102 instead?

Reply:
Resolved offline.